docs/docs/0.3.2/documentation/README.md

```diff
@@ -390,10 +390,10 @@ spec:
       ...
 ```
 
-::: tip
-When integrating [Kong for Kubernetes](https://github.com/Kong/kubernetes-ingress-controller) with Kuma you have to annotate every `Service` that you want to pass traffic to with [`ingress.kubernetes.io/service-upstream=true`](https://github.com/Kong/kubernetes-ingress-controller/blob/master/docs/references/annotations.md#ingresskubernetesioservice-upstream) annotation.
-Otherwise Kong will do the load balancing which unables Kuma to do the load balancing and apply policies. 
-:::
+The optimal gateway in Kubernetes mode would be Kong. You can use [Kong for Kubernetes](https://github.com/Kong/kubernetes-ingress-controller) to implement authentication, transformations, and other functionalities across Kubernetes clusters with zero downtime. When integrating [Kong for Kubernetes](https://github.com/Kong/kubernetes-ingress-controller) with Kuma you have to annotate every `Service` that you want to pass traffic to with [`ingress.kubernetes.io/service-upstream=true`](https://github.com/Kong/kubernetes-ingress-controller/blob/master/docs/references/annotations.md#ingresskubernetesioservice-upstream) annotation. Otherwise Kong will do the load balancing which enables Kuma to do the load balancing and apply policies. 
```

Comment:
> Otherwise Kong will do the load balancing which enables Kuma to do the load balancing and apply policies. 

The meaning of this sentence was reverted:
* previously, it was `which UNABLES Kuma to do the load balancing`
* now, it is `which ENABLES Kuma to do the load balancing`

The correct version is the former one.

Reply:
spell check auto corrected this one haha, updated back to unables